### ISSUE TITLE
loss does not decrease (help )

### ISSUE THREAD

**Issue by molo32** [pytorch/pytorch#52879]
I am modifying two projects, one is a clip that optimizes the latent code of stylegan, another is acface that generates a code when passing a face to it.
if the faces are similar, the distance between the acface codes is small.

then

I have two variables embeds_s and embeds
embeds_s: is the code generated by arcface of the face of a jpg file
embeds: is the code generated by arcface of the face synthesized by stylegan

I calculate the error like this
```
ident_loss = sum (((embeds_s-embeds) ** 2) .reshape (512))
 loss = ident_loss
 optimizer.zero_grad ()
 loss.backward ()
 optimizer.step ()
```

  but the loss error does not decrease, however it does decrease if I add the cliploss.
clip loss is just a value like ident_loss is,
because with clip_loss it decreases but not with ident_loss?
 what am I doing wrong?:

here I leave the colab notebook so you can see

 [https://colab.research.google.com/drive/1Cg4dhqWEjkEgoQap-2W6jnO2P6ibOTdc#scrollTo=x0VHBi9NWglq](url)

cc @vincentqb

**Comment by skyline75489** [comment]
Github is not a general technical support forum. The issues are supposed to be about the development of PyTorch *itself*. Other issues are likely to be ignored. Please use [the forum](https://discuss.pytorch.org/) instead.

**Comment by vincentqb** [comment]
As recommended above, this issue is better discussed on [the forum](https://discuss.pytorch.org/). I recommend that you open a question there. I will close this issue, but please feel free to re-open if you believe this does not address your issue.
